src/custom-element.js

```diff
@@ -519,14 +587,22 @@ function createBaseCustomElementClass(win) {
     }
 
     /**
-     * Ensure that element is eagerly loaded.
+     * Ensure that the element is eagerly loaded.
      *
      * @param {number=} opt_parentPriority
      * @return {!Promise}
      * @final
      */
     ensureLoaded(opt_parentPriority) {
```

Comment:
You're right about the `Owners` use cases here. However, they are still valid for the time being. I hope to get rid of them soon enough, but in the meantime they are still used.

One more use case for this API are cases like `amp-animation`: it doesn't want to animate unloaded elements. Otherwise an animation would look broken.

I think where we could take it eventually is something like this:

```
// I need element to be fully loaded:
new Promise(resolve => {
  element.build(); // force build
  if (element.readyState == 'complete') {
    resolve();
  } else {
    element.loading = 'eager';
    element.onload = resolve;
  }
});
```

This is roughly what `ensureLoaded` is doing right now. I'm not sure we're entirely ready to move in this direction, tbh.